test/mysql/ConnectionTest.chpl

```diff
@@ -0,0 +1,38 @@
+use DatabaseCommunicator;
+use DatabaseCommunicator.DatabaseCommunicationObjects.ErrorTypes;
+use DatabaseCommunicator.QueryBuilder;
+use UnitTest;
+use MySQL;
+
+proc configConnectionTestNormal(test: borrowed Test) throws {
+  var conHandler = ConnectionHandler.ConnectionHandlerWithConfig(MySQLConnection, "dbconfig.toml");
+  conHandler.close();
+}
+
+proc configConnectionTestDifferentRootElem(test: borrowed Test) throws {
+  // different root element name in TOML file
+  var conHandler = ConnectionHandler.ConnectionHandlerWithConfig(MySQLConnection, "dbconfig2.toml", "dbconf");
+  conHandler.close();
+}
+
+// TODO: Figure out a way to run all the follwing tests in one go
```

Comment:
This shouldn't be the case. Let's discuss this offline. This may be a bug in the UnitTest library. 😅